lib/rollbax/item.ex

```diff
@@ -33,4 +38,35 @@ defmodule Rollbax.Item do
       Map.put(data, "custom", meta)
     end
   end
+
+  defp language do
+    "Elixir " <> System.version
+  end
+
+  defp framework do
+    :erlang.system_info(:system_version)
+    |> to_string
+    |> String.strip
+  end
+
+  defp platform do
+    type = :os.type
+    |> Tuple.to_list
+    |> Stream.map(&Atom.to_string/1)
+    |> Stream.map(&String.capitalize/1)
+    |> Enum.join("/")
+
+    version = :os.version |> Tuple.to_list |> Enum.join(".")
+    type <> " " <> version
+  end
+
+  defp notifier do
+    %{
+      "name" => "Rollbax",
+      "version" => case :application.get_key(:rollbax, :vsn) do
```

Comment:
We can avoid `:application.get_key` "dance" by accessing `Mix.Project.config`:
```elixir
unquote(Mix.Project.config[:version])
```
Please note that we need `unquote` to resolve it during compile time and not depend on Mix in runtime.